### ISSUE TITLE
3.2.0 release

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#1015]
Can be labelled **ready** when all other issues on this milestone are closed.

- [x] run code analysis in VS in *Release* mode and address violations (send a regular PR which must be merged before continuing)
- [x] if necessary, change `VERSION_SUFFIX` in [AppVeyor](https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/settings) to  "" and initiate a build
- [x] check [build](https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/build/158)
- edit draft release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases):
    - [x] ensure completeness of release notes, including non-owner contributors, if any (move release notes forward from any pre-releases to the current release)
    - [x] attach main nupkg and/or analyzer nupkg, whichever have content to release
    - [x] publish the release
- [x] push nupkgs to NuGet using the Deploy option on the AppVeyor build
- [x] de-list pre-release or superseded buggy NuGet packages if present
- [x] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [x] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
- [x] post a link to the GitHub Release in each issue in this milestone, with thanks to contributors
- ~~run `rake next_version[new_version]` to~~
    - [x] create a pull request that changes the version in CommonAssemblyInfo.cs to the expected version (of form _xx.yy.zz_)
    - [x] create a new draft GitHub Release
    - [x] create a new milestone for the next release
    - [x] create a new issue (like this one) for the next release, adding it to the new milestone
- [x] close this milestone

**Comment by adamralph** [comment]
FYI removed the link to #1014 from the checklist (copypasta artifact from #986).

**Comment by blairconrad** [comment]
Oh, thanks. That sounds like the kind of error I'd make! :blush:

**Comment by blairconrad** [comment]
It's been over 3 weeks since a release, and we have some useful content. What do you think about putting 3.2.0 out there?
#1030 is the only issue we have in flight right now (and I think it looks good - @adamralph, are you still waiting for changes?), but I don't mind releasing either with or without it.
Oh! Unless we want to take a stab at #1039 before the release.

Either way, I'm probably free Wednesday morning.

**Comment by thomaslevesque** [comment]
Yes, I think we're ready to release once @1030 is merged.
#1039 would be a nice to have, if we can make it work.

**Comment by blairconrad** [comment]
Oh, @thomaslevesque. You just pinged poor user 1030! 
I nearly did the same when I typed #1030 in my original comment.

**Comment by blairconrad** [comment]
Okay. #1030 is merged.
Do we trust #1040 enough to at least merge it, assuming it will do no harm?

**Comment by adamralph** [comment]
I'm 👍 to release 3.2.0.

**Comment by thomaslevesque** [comment]
> Do we trust #1040 enough to at least merge it, assuming it will do no harm?

What's the worst that could happen? It just won't deploy, and we'll have to do it manually. No big deal.

**Comment by blairconrad** [comment]
That's my thought too.

**Comment by blairconrad** [comment]
Marking **ready** since everything that we think we want to go in is closed.

**Comment by blairconrad** [comment]
I'm free on the morning of 5 April, if you want me to release. Or I don't mind if someone else wants the glory.

**Comment by thomaslevesque** [comment]
> I'm free on the morning of 5 April, if you want me to release. Or I don't mind if someone else wants the glory.

Please do! I probably won't have the time to do it tomorrow.

**Comment by adamralph** [comment]
:shipit:

**Comment by blairconrad** [comment]
Shipitting.

**Comment by blairconrad** [comment]
https://twitter.com/fakeiteasyfx/status/849565562296565761
